R/OsModel.R

```diff
@@ -0,0 +1,171 @@
+#' Title - TODO
+#'
+#' Description - TODO
+#'
+#' @slot stan TODO
+#' @slot templated Logical. Defines whether the OsModel object needs to be completed
+#' @export
+OsModel <- setClass(
+    Class = "OsModel",
+    representation = list(
+        stan = "StanModule",
+        templated = "logical"
+    )
+)
+
+
+#'  Log-logistic overall survival
+#'
+#' Description - Log-logistic overall survival model object creator
+#'
+#' @export
+LogLogisticOs <- setClass(
+    Class = "LogLogisticOs",
+    contains = "OsModel"
+)
+
+
+#' LogLogisticModule
+#'
+#' Log logistic module helper function.
+#' @param functions
+#' @param data
+#' @param parameters
+#' @param transformed_parameters
+#' @param generated_quantities
+#' @param priors
+#' @param inits
+#' @param templated
+#' @export
+LogLogisticModule <- function(functions = "os_functions.stan",
```

Comment:
I think these will need to point to the system files  as they won't be "local" when this is installed as a package

Reply:
This function should do the work, no ? 

https://github.com/Genentech/jmpost/blob/b69df60562ba6f07823b8218da88a59b2fc2b5cd/R/stan_module.R#L32-L42

Reply:
O yes you are right apologies